meos/include/meos.h

```diff
@@ -353,6 +356,9 @@ extern bool meos_set_intervalstyle(char *newval, int extra);
 extern char *meos_get_datestyle(void);
 extern char *meos_get_intervalstyle(void);
 
+extern gsl_rng *gsl_get_generation_rng(void);
+extern gsl_rng *gsl_get_aggregation_rng(void);
```

Comment:
Should these functions be exposed in `meos.h`?

Reply:
These functions are used for example in
https://github.com/estebanzimanyi/MobilityDB/blob/global/meos/src/point/tpoint_datagen.c#L258
https://github.com/estebanzimanyi/MobilityDB/blob/global/meos/src/general/skiplist.c#L124